traitlets/config/application.py

```diff
@@ -583,21 +584,26 @@ def _load_config_files(cls, basefilename, path=None, log=None, raise_config_file
                                 " {1} has higher priority: {2}".format(
                                 filename, loader.full_filename, json.dumps(collisions, indent=2),
                             ))
-                    yield config
+                    yield (config, loader.full_filename)
```

Comment:
FWIW, this is a private API call that was indeed used by IPython until 5.5, so this patch-release of traitlets will break IPython <5.5 (5.8 is the last supporting Python 2 and 7.8 is current). The API that called this was somewhat  rarely used (mainly in `IPython.embed()`).

Reply:
https://github.com/ipython/ipython/pull/10676

Reply:
Good information @minrk - thank you.